### PR TITLE
Added damper to button

### PR DESCRIPTION
**Pull request by fooker**

### CODE REVIEW

**Comment by mweinelt** [comment]
Waiting for Feedback from @MyIgel.

**Comment by MyIgel** [comment]
It "works" as in it does not flicker but it should damper for < 1s and its actually buggy: sometimes it works but sometimes i have to push the button multiple times to trigger a change (somtimes up to 10 seconds after trying it multiple times)

File: framework/features/Button.h [hunk 1]
```diff
@@ -35,6 +36,8 @@ class Button : public Feature<name> {
 
 private:
     virtual void onInterrupt()  {
+        LOG.log("!!!!!!!!!!!!!!!!!!!!!");
```
Comment (MyIgel) [inline framework/features/Button.h]: Moar -vvv :D

**Comment by mweinelt** [comment]
I don't see the dampening effect on a Socket.

Tested 17a9b1e on Sonoff S20.

**Comment by MyIgel** [comment]
Jup, now its works as I expected it :)

**Pull request merged**